modules/engine/engine.cpp

```diff
@@ -455,7 +455,12 @@ static cell AMX_NATIVE_CALL set_view(AMX *amx, cell *params) {
 			g_pFunctionTable_Post->pfnPlayerPostThink=PlayerPostThink_Post;
 
 			plinfo[iIndex].iViewType = CAMERA_3RDPERSON;
-			pNewCamera = CREATE_NAMED_ENTITY(MAKE_STRING("info_target"));
+
+			if (!(pNewCamera = CREATE_NAMED_ENTITY(MAKE_STRING("info_target"))))
```

Comment:
Assertion should be better, it is not normal that entity is `nullptr`. Or native error.